source/runtime_gui.cpp

```diff
@@ -2266,6 +2266,16 @@ void reshade::runtime::draw_gui_settings()
 		modified |= ImGui::Checkbox(_("Save before and after images"), &_screenshot_save_before);
 #endif
 		modified |= ImGui::Checkbox(_("Save separate image with the overlay visible"), &_screenshot_save_gui);
+		//
+		// TODO: Add string table entries for this stuff...
```

Comment:
Localization works by wrapping strings in `_("...")` and then running `tools\create_language_template.ps1` with the various language identifiers ReShade has localization for (e.g. `tools\create_language_template.ps1 en-US`). That will update the language resources with the new strings, which can then be translated. But I can do that in post.